### ISSUE TITLE
Update add-on branding 

### ISSUE THREAD

**Issue by PerfectSlayer** [PerfectSlayer/scrollupfolder#41]
As the add-on being updated for Web Extensions (#40) , I can no more use the scroll event.
So I guess I have to rename the application in order to follow the first rule of "[Creating an appealing add-on listing](https://developer.mozilla.org/en-US/Add-ons/Listing)": 

> so it gives the user a good indication of what your add-on does

And it will no more scroll up!
Moreover, I guess I need to update the icon (quite old now). So any name and icon idea is welcome!

Personally I was thinking about:
- Browse parent folders
- Browse up folders
- Browse folders
- Navigate parent folders
- Navigate up folders
- Navigate folders

**Comment by Gitoffthelawn** [comment]
I like "Navigate folders", or perhaps "Navigate site folders".

**Comment by Gitoffthelawn** [comment]
BTW, I do wonder if the scroll API will be re-introduced in WebExtensions sometime.

**Comment by PerfectSlayer** [comment]
"Navigate site folders"? Not bad!

I don't think it will. There is actually no way to mess with existing UI elements. You can only [create new predefined elements](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/user_interface) (like button, sidebar, context menu).

**Comment by PerfectSlayer** [comment]
I just add the "Fold up!" name proposition to the survey. It looks nice! 😄
But the survey is anonymous so I don't know who proposed it 😅

**Comment by Gregory-K** [comment]
As a "new user" of the web-extension version, I don't have much of a saying.  

I just want to suggest that the "scroll" in the name is a little misleading. I don't know if any more functions will be added at a later point in time but the current functionality is often titled (icon hover) as plain "up" in most OS GUIs. "Scroll" is different function and the only application I can thing of is on one-page websites. I like the "Fold up!" better.
The name (my opinion) should also be small. Therefore, I am inclined towards "UpFolder", "FolderUp"/"Fold Up!", "GoUp", "GoUp Folder", "UpDir", "DirUp", don't know, something like these.

The icon can be something like this draft I made (included in a push request with corresponding gray url icons)
![foldup](https://user-images.githubusercontent.com/5237731/32922628-c4a18ede-cb3b-11e7-8f39-056760c35eb8.png)

Of course, all these are mere suggestions/thoughts, not trying to overstep in any way.

**Comment by Gitoffthelawn** [comment]
@Gregory-K You're not overstepping in any way!  All opinions are welcome.

I'm trying to like *Fold Up!*, but I just don't... I think the reason is because I don't ever view folders as folding or unfolding... especially when going up or down a level in the tree.

That said, I appreciate your energy and completely respect your opinion.  :)

**Comment by Gregory-K** [comment]
Ah, me too, I just thought not to over-over-step :) (till last min I misread it as "Folder"). I see directories as directory trees and thats my unix/dos heritage but most users now don't even have a file-manager on their smart devices... The most accurate description in this case, I think, is "browse parent folders" and it is still a metaphor for dynamic websites that use clean URLs. Thanks that's a common practice nowadays (for the user mostly). I also drifted to product marketing ways of thinking. It may have a bigger name, the add-on is not a company, the theoretical "logo" doesn't have to embed the title .

What ever you (plural) decide.

**Comment by PerfectSlayer** [comment]
Hey! It's nice to see a discussion here. I mean, with someone else than me 😄 

@Gregory-K  Exactly, the "scroll" does not mean anything now due to WebExtensions. It was the main feature of the previous versions but it will not come back.

To help us to choose a name, I will include the (current) results of the survey:
![image](https://user-images.githubusercontent.com/1766222/32982874-900df0d0-cc8b-11e7-880c-e3472b236903.png)

There is a majority who wants to keep the original name. But only 6 or 7 only vote to keep the original name as the only option (the others vote for other names too). Here are some comment I received:

> Hello, I've always found this a very useful add on to Firefox. BTW... I would keep the name "Scroll Up Folder". It just describes add-on accurately so other's will know what it does when searching for this functionality. Thanks for actively developing this add-on.

> I'm using Scroll Up Folder since years now. It's pretty useful and IMO the name now is a 'brand'. And, even more good, the name represent what the add-on is doing. Maybe you do not like the name of your tool anymore but if you rename it you might write the former name into the description. That would be the perfect gobbledygook then. Muchas Greetings from Munich

Obviously, they should have received the survey before testing the WebExtension version. But I like the idea of the _brand_. Thanks to them. I hope the other participants noticed the scroll feature do no more work before they answer the survey (those who want to keep the original name in particular).

So, in summary, we have:
1. Fold up!
2. Navigate site folders
3. Navigate up folders

"Fold up!" is very catchy but does it really mean something? Does _it gives the user a good indication of what your add-on does_ like the AMO recommendations ask? :anguished:

**Comment by Gitoffthelawn** [comment]
I think "Fold Up!" is fun, but confusing.  I like "Navigate folders", "Navigate site folders", and "Navigate up folders".

I think the tooltip might need an adjustment to reflect one of those... probably just "Navigate folders" for the tooltip, regardless of the extension name chosen.

**Comment by Gregory-K** [comment]
In my opinion, there are three ways to name it:
a. promote with a funky catchy name like "fold up". A "folder" is a folder and it can fold-up (https://en.wikipedia.org/wiki/Directory_(computing))
b. promote with a short direct name like "go up" (or a little bigger "go up folder"). The majority of browsers and file explorers use the same terminology (go back, go forward, go up, back to, forward to, up to etc)
c. just describe the "product/add-on" in the title and subtitle/description like "browse parent folders, navigate up folders etc. I have to mention that any "navigate" or "browse" name without the "up" part gives me the expectation of a site-map/site-menu.

As I said, it is just an extension and a free one to dedicate a lot of brain cycles promoting it but if you want to give that extra thought it is either "a" or "b".
I give a plus to "b" for keeping a consistent browser terminology. (I know it isn't on the shortlist).
"Fold Up" is also a good hipster web v3 name. In the case of "Fold Up" I suggest the icon's alt text to be "go up" for the sake of semantics ^.
Descriptions for both of them can be meticulously explanatory.

*Food for thought*
What would you write into the search field at "addons.mozilla.org" hopping to find exactly what this add-on does? (anything with "up" and "folder" is sorting this add-on already high)
I don't know the algorithm at "addons.mozilla.org" on sorting. Are there tags and what is their weight? The description or the title has more weight?
